drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/handlers/GuidedDecisionTableOptions.java

```diff
@@ -41,11 +42,15 @@
     @UiField
     CheckBox chkUseWizard;
 
+    @UiField(provided = true)
+    HitPolicySelector hitPolicySelector;
+
     private boolean isUsingWizard = false;
 
     private GuidedDecisionTable52.TableFormat tableFormat = GuidedDecisionTable52.TableFormat.EXTENDED_ENTRY;
 
     public GuidedDecisionTableOptions() {
+        hitPolicySelector = new HitPolicySelector( IOC.getBeanManager().lookupBean( HitPolicySelectorView.class ).getInstance() );
```

Comment:
```GuidedDecisionTableOptions``` is injected into ```NewGuidedDecisionTableHandler``` and hence you should not need to instantiate the View in this way. (i.e. you should be able to inject it too). At the very least you'll need to handle destruction of the view instance somewhere/somehow.